Automattic/jetpack
# Check user email value when deciding to show subscription options

In Verbum users can login via FB without sharing their email.
When this happens the subscription option toggles don't make much sense, and result in bad payload.
This PR disables/hides them for this case.

Fixes #
142-gh-Automattic/verbum

## Proposed changes:

* Consider user email value when calculating `hasSubOptions`

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Sync this PR
* Open a post on your site
* For commenting use FB without sharing your email
* The subscription options should be hidden
* Login via WP account. 
* The subscription options should be visible
